AtReconstruction/AtReconstructionLinkDef.h

```diff
@@ -37,6 +37,10 @@
 #pragma link C++ class SampleConsensus::AtEstimator - !;
 #pragma link C++ enum SampleConsensus::Estimators;
 
+#pragma link C++ class AtPatternModification + ;
+#pragma link C++ class AtBraggCurveFinder + ;
+#pragma link C++ class AtPatternModificationTask + ;
```

Comment:
All of these should be -!, not + I think. None of them are data structures so be written to disk.